src/Microsoft.AspNet.Mvc.ViewFeatures/Rendering/MvcForm.cs

```diff
@@ -8,11 +8,18 @@
 
 namespace Microsoft.AspNet.Mvc.Rendering
 {
+    /// <summary>
+    /// Represents an HTML form element in an MVC view.
```

Comment:
Ok, I guess we've not been using that term, so I'll keep up with that tradition.

> An HTML form element in an MVC view.